### ISSUE TITLE
Module.bazel support

### ISSUE THREAD

**Issue by XaveScor** [fremtind/rules_vitest#19]
Thank you for your port.
I migrate our project to bzlmod and have the error below: 
```
INFO: Invocation ID: fcbed8d9-1a5e-44c8-b9cd-760862b4986e
ERROR: Error computing the main repository mapping: in module dependency chain <root> -> fremtind_rules_vitest@0.1.0: module not found in registries: fremtind_rules_vitest@0.1.0
```
 
Also, I need help finding the dep in the main repo: https://registry.bazel.build/search?q=fremtind_rules_vitest. I think it is the cause why bzlmod is not working.

**Comment by hjellek** [comment]
Thanks for your issue! 

We are awaiting the release of rules_js 2.0.0 before releasing the ruleset on the bazel central registry.

See https://github.com/fremtind/rules_vitest/issues/14 for more details.

**Comment by hjellek** [comment]
`rules_vitest v0.2.0` is now merged in BCR with `rules_js 2.0` support!

https://github.com/bazelbuild/bazel-central-registry/pull/2609
